### ISSUE TITLE
rocRAND/hipRAND 5.4.0 packages are missing symlinks

### ISSUE THREAD

**Issue by upsj** [ROCmSoftwarePlatform/rocRAND#315]
**Describe the bug**
The packages for hipRAND and rocRAND provided for ROCm 5.4.0 at https://repo.radeon.com/ are missing symlinks for `hiprand-fortran-config.cmake` and `rocrand-fortran-config.cmake`

**To Reproduce**
Install `rocrand`, try to find rocrand using CMake `find_package(rocrand)` with `ROCRAND_PATH` set to `/opt/rocm-5.4.0/rocrand`
For the installation, I used
```
wget https://repo.radeon.com/amdgpu-install/5.4/ubuntu/jammy/amdgpu-install_5.4.50400-1_all.deb \
    && apt-get install -y --no-install-recommends ./amdgpu-install_5.4.50400-1_all.deb \
    && apt-get update && apt-get install -y --no-install-recommends \
    rocm-dev hipblas-dev hipfft-dev hipsparse-dev rocfft-dev rocrand-dev rocsolver-dev rocthrust-dev roctracer-dev
```

**Expected behavior**
The find module succeeds.

**Log-files**
```
CMake Error at /opt/rocm-5.4.0/rocrand/lib/cmake/rocrand-config.cmake:92 (include):
  include could not find requested file:

    /opt/rocm-5.4.0/rocrand/lib/cmake/rocrand-fortran-config.cmake
Call Stack (most recent call first):
  cmake/hip.cmake:176 (find_package)
  CMakeLists.txt:101 (include)
```

**Workaround**
Add the symlinks via
```
cd /opt/rocm-5.4.0/hiprand/lib/cmake/ && ln -s ../../../lib/cmake/hiprand/hiprand-fortran-config.cmake
cd /opt/rocm-5.4.0/rocrand/lib/cmake/ && ln -s ../../../lib/cmake/rocrand/rocrand-fortran-config.cmake
```

**Environment**
<details>

```
=== environment


=== date
Sat Feb  4 23:41:26 UTC 2023


=== Linux Kernel
Linux 2e88cbd9c48a 4.18.0-408.el8.x86_64 #1 SMP Mon Jul 18 17:42:52 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux


=== rocm-smi

======================= ROCm System Management Interface =======================
================================= Concise Info =================================
GPU  Temp (DieEdge)  AvgPwr  SCLK    MCLK    Fan     Perf  PwrCap  VRAM%  GPU%  
0    30.0c           22.0W   808Mhz  350Mhz  20.78%  auto  250.0W    0%   0%    
================================================================================
============================= End of ROCm SMI Log ==============================


HIP version  : 5.4.22801-aaa1e3d8

== hipconfig
HIP_PATH     : /opt/rocm-5.4.0/hip
ROCM_PATH    : /opt/rocm-5.4.0
HIP_COMPILER : clang
HIP_PLATFORM : amd
HIP_RUNTIME  : rocclr
CPP_CONFIG   :  -D__HIP_PLATFORM_HCC__= -D__HIP_PLATFORM_AMD__= -I/opt/rocm-5.4.0/hip/include -I/opt/rocm-5.4.0/llvm/bin/../lib/clang/15.0.0 -I/opt/rocm-5.4.0/hsa/include

== hip-clang
HSA_PATH         : /opt/rocm-5.4.0/hsa
HIP_CLANG_PATH   : /opt/rocm-5.4.0/llvm/bin
AMD clang version 15.0.0 (https://github.com/RadeonOpenCompute/llvm-project roc-5.4.0 22465 d6f0fe8b22e3d8ce0f2cbd657ea14b16043018a5)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /opt/rocm-5.4.0/llvm/bin
AMD LLVM version 15.0.0git
  Optimized build.
  Default target: x86_64-unknown-linux-gnu
  Host CPU: znver1

  Registered Targets:
    amdgcn - AMD GCN GPUs
    r600   - AMD GPUs HD2XXX-HD6XXX
    x86    - 32-bit X86: Pentium-Pro and above
    x86-64 - 64-bit X86: EM64T and AMD64
hip-clang-cxxflags :  -isystem "/opt/rocm-5.4.0/llvm/lib/clang/15.0.0/include/.." -isystem /opt/rocm-5.4.0/hsa/include -isystem "/opt/rocm-5.4.0/hip/include" -O3 --rocm-path=/opt/rocm-5.4.0
hip-clang-ldflags  :  -L"/opt/rocm-5.4.0/hip/lib" -O3 -lgcc_s -lgcc -lpthread -lm -lrt

=== Environment Variables
PATH=/opt/intel/oneapi/dpcpp-ct/2023.0.0/bin:/opt/intel/oneapi/vtune/2023.0.0/bin64:/opt/intel/oneapi/mkl/2023.0.0/bin/intel64:/opt/intel/oneapi/compiler/2023.0.0/linux/lib/oclfpga/bin:/opt/intel/oneapi/compiler/2023.0.0/linux/bin/intel64:/opt/intel/oneapi/compiler/2023.0.0/linux/bin:/opt/rocm-5.4.0/bin:/root/spack-env/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/root/spack/bin
CUDACXX=/root/spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.3.0/cuda-11.8.0-bf5ocuyzznq57tji4taz2m3a6yuuvgoa/bin/nvcc
HIPCC_COMPILE_FLAGS_APPEND=--rocm-path=/opt/rocm-5.4.0
HIP_CLANG_PATH=/opt/rocm-5.4.0/llvm/bin
LD_LIBRARY_PATH=/opt/intel/oneapi/tbb/2021.8.0/lib/intel64/gcc4.8:/opt/intel/oneapi/mkl/2023.0.0/lib/intel64:/opt/intel/oneapi/compiler/2023.0.0/linux/lib:/opt/intel/oneapi/compiler/2023.0.0/linux/lib/x64:/opt/intel/oneapi/compiler/2023.0.0/linux/lib/oclfpga/host/linux64/lib:/opt/intel/oneapi/compiler/2023.0.0/linux/compiler/lib/intel64_lin
HIP_COMPILER=clang
HSA_PATH=/opt/rocm-5.4.0/hsa
HIP_PATH=/opt/rocm-5.4.0/hip
HIP_PLATFORM=amd
HIP_DEVICE_LIB_PATH=/opt/rocm-5.4.0/amdgcn/bitcode

== Linux Kernel
Hostname     : 2e88cbd9c48a
Linux 2e88cbd9c48a 4.18.0-408.el8.x86_64 #1 SMP Mon Jul 18 17:42:52 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.1 LTS
Release:        22.04
Codename:       jammy



=== rocminfo
ROCk module is loaded
=====================    
HSA System Attributes    
=====================    
Runtime Version:         1.1
System Timestamp Freq.:  1000.000000MHz
Sig. Max Wait Duration:  18446744073709551615 (0xFFFFFFFFFFFFFFFF) (timestamp count)
Machine Model:           LARGE                              
System Endianness:       LITTLE                             

==========               
HSA Agents               
==========               
*******                  
Agent 1                  
*******                  
  Name:                    AMD Ryzen Threadripper 1920X 12-Core Processor
  Uuid:                    CPU-XX                             
  Marketing Name:          AMD Ryzen Threadripper 1920X 12-Core Processor
  Vendor Name:             CPU                                
  Feature:                 None specified                     
  Profile:                 FULL_PROFILE                       
  Float Round Mode:        NEAR                               
  Max Queue Number:        0(0x0)                             
  Queue Min Size:          0(0x0)                             
  Queue Max Size:          0(0x0)                             
  Queue Type:              MULTI                              
  Node:                    0                                  
  Device Type:             CPU                                
  Cache Info:              
    L1:                      32768(0x8000) KB                   
  Chip ID:                 0(0x0)                             
  ASIC Revision:           0(0x0)                             
  Cacheline Size:          64(0x40)                           
  Max Clock Freq. (MHz):   3500                               
  BDFID:                   0                                  
  Internal Node ID:        0                                  
  Compute Unit:            24                                 
  SIMDs per CU:            0                                  
  Shader Engines:          0                                  
  Shader Arrs. per Eng.:   0                                  
  WatchPts on Addr. Ranges:1                                  
  Features:                None
  Pool Info:               
    Pool 1                   
      Segment:                 GLOBAL; FLAGS: FINE GRAINED        
      Size:                    65556416(0x3e84fc0) KB             
      Allocatable:             TRUE                               
      Alloc Granule:           4KB                                
      Alloc Alignment:         4KB                                
      Accessible by all:       TRUE                               
    Pool 2                   
      Segment:                 GLOBAL; FLAGS: KERNARG, FINE GRAINED
      Size:                    65556416(0x3e84fc0) KB             
      Allocatable:             TRUE                               
      Alloc Granule:           4KB                                
      Alloc Alignment:         4KB                                
      Accessible by all:       TRUE                               
    Pool 3                   
      Segment:                 GLOBAL; FLAGS: COARSE GRAINED      
      Size:                    65556416(0x3e84fc0) KB             
      Allocatable:             TRUE                               
      Alloc Granule:           4KB                                
      Alloc Alignment:         4KB                                
      Accessible by all:       TRUE                               
  ISA Info:                
*******                  
Agent 2                  
*******                  
  Name:                    gfx906                             
  Uuid:                    GPU-38ce408172dc76e5               
  Marketing Name:          AMD Radeon VII                     
  Vendor Name:             AMD                                
  Feature:                 KERNEL_DISPATCH                    
  Profile:                 BASE_PROFILE                       
  Float Round Mode:        NEAR                               
  Max Queue Number:        128(0x80)                          
  Queue Min Size:          64(0x40)                           
  Queue Max Size:          131072(0x20000)                    
  Queue Type:              MULTI                              
  Node:                    1                                  
  Device Type:             GPU                                
  Cache Info:              
    L1:                      16(0x10) KB                        
    L2:                      8192(0x2000) KB                    
  Chip ID:                 26287(0x66af)                      
  ASIC Revision:           1(0x1)                             
  Cacheline Size:          64(0x40)                           
  Max Clock Freq. (MHz):   1801                               
  BDFID:                   17664                              
  Internal Node ID:        1                                  
  Compute Unit:            60                                 
  SIMDs per CU:            4                                  
  Shader Engines:          4                                  
  Shader Arrs. per Eng.:   1                                  
  WatchPts on Addr. Ranges:4                                  
  Features:                KERNEL_DISPATCH 
  Fast F16 Operation:      TRUE                               
  Wavefront Size:          64(0x40)                           
  Workgroup Max Size:      1024(0x400)                        
  Workgroup Max Size per Dimension:
    x                        1024(0x400)                        
    y                        1024(0x400)                        
    z                        1024(0x400)                        
  Max Waves Per CU:        40(0x28)                           
  Max Work-item Per CU:    2560(0xa00)                        
  Grid Max Size:           4294967295(0xffffffff)             
  Grid Max Size per Dimension:
    x                        4294967295(0xffffffff)             
    y                        4294967295(0xffffffff)             
    z                        4294967295(0xffffffff)             
  Max fbarriers/Workgrp:   32                                 
  Pool Info:               
    Pool 1                   
      Segment:                 GLOBAL; FLAGS: COARSE GRAINED      
      Size:                    16760832(0xffc000) KB              
      Allocatable:             TRUE                               
      Alloc Granule:           4KB                                
      Alloc Alignment:         4KB                                
      Accessible by all:       FALSE                              
    Pool 2                   
      Segment:                 GROUP                              
      Size:                    64(0x40) KB                        
      Allocatable:             FALSE                              
      Alloc Granule:           0KB                                
      Alloc Alignment:         0KB                                
      Accessible by all:       FALSE                              
  ISA Info:                
    ISA 1                    
      Name:                    amdgcn-amd-amdhsa--gfx906:sramecc+:xnack-
      Machine Models:          HSA_MACHINE_MODEL_LARGE            
      Profiles:                HSA_PROFILE_BASE                   
      Default Rounding Mode:   NEAR                               
      Default Rounding Mode:   NEAR                               
      Fast f16:                TRUE                               
      Workgroup Max Size:      1024(0x400)                        
      Workgroup Max Size per Dimension:
        x                        1024(0x400)                        
        y                        1024(0x400)                        
        z                        1024(0x400)                        
      Grid Max Size:           4294967295(0xffffffff)             
      Grid Max Size per Dimension:
        x                        4294967295(0xffffffff)             
        y                        4294967295(0xffffffff)             
        z                        4294967295(0xffffffff)             
      FBarrier Max Size:       32                                 
*** Done ***             


=== lspci VGA
08:00.0 VGA compatible controller: NVIDIA Corporation GP102 [TITAN X] (rev a1)
45:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Vega 20 [Radeon VII] (rev c1)
```

</details>

**Comment by cgmb** [comment]
I'm not certain, but I believe this is the issue that was addressed by https://github.com/RadeonOpenCompute/rocm-cmake/pull/107. If so, then you should see a fix in ROCm 5.5.

**Comment by doctorcolinsmith** [comment]
Issue was resolved with PR 107.
